baggage/utils.py

```diff
@@ -0,0 +1,95 @@
+import math
+import itertools
+from baggage.models import Room, Bag, BAG_ADDED
+# from escpos.printer import Usb
+# from escpos.escpos import EscposIO
+# from django.contrib.staticfiles import finders
+# import time
+# import datetime
```

Comment:
Delete commented imports

Reply:
Those will be deleted but not now, I'm still testing that...

Reply:
Add comment to remember to delete on PR and code as `todo`

Reply:
Added on #54.